src/components/__tests__/__snapshots__/ExternalTooltip.spec.jsx.snap

```diff
@@ -0,0 +1,68 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`initial state renders nothing without crashing 1`] = `null`;
+
+exports[`initial state renders without crashing 1`] = `
+<section
+  className="tooltip-container u-clearfix"
+>
+  <p
+    className="a-micro-copy"
+  >
+    <span>
+      Date interval: 
+      ext tip title
```

Comment:
Maybe the title label also needs to be passed in?  Or just make the title dump a single string

Reply:
Good idea.  Sometimes that Date Interval text isn't applicable, for instance a Single Day